### PR TITLE
Build system: simplify build script

### PR DESCRIPTION
**Pull request by saper**
- simplify Travis build script,
  remove bash-specific code

### CODE REVIEW

**Comment by saper** [comment]
The primary problem solved here is that on FreeBSD (and maybe MacOSX) we cannot just say "-lstdc++" when using clang-based toolchain.

File: Makefile [hunk 1]
```diff
@@ -49,9 +49,9 @@ else
 	LDFLAGS  += -std=c++0x
 endif
 
-ifneq "$(SASS_LIBSASS_PATH)" ""
-	CFLAGS   += -I $(SASS_LIBSASS_PATH)
-	CXXFLAGS += -I $(SASS_LIBSASS_PATH)
+ifneq "$(LIBSASS_SRC_DIR)" ""
```
Comment (xzyfer) [inline Makefile]: What's the justification for changing this variable?

File: script/ci-build-libsass [hunk 2]
```diff
@@ -1,89 +0,0 @@
-#!/bin/bash
```
Comment (xzyfer) [inline script/ci-build-libsass]: What's the justification for moving this file? This change would appear to be inconsequential to this PR.

**Comment by xzyfer** [comment]
@saper please fix the merge conflicts

**Comment by saper** [comment]
@mgreter @am11 can you try building https://github.com/saper/sassc/commit/828d7da85bd355baf9bd3ae2297781889b09b70e changing https://github.com/saper/sassc/blob/828d7da85bd355baf9bd3ae2297781889b09b70e/script/ci-build-libsass#L37 to `-j5` ? I think parallel builds should work with mingw32 now as well...

**Comment by saper** [comment]
Travis CI should be ok once https://github.com/sass/libsass/pull/1540 gets into libsass master.

**Comment by xzyfer** [comment]
Rebuilding Travis now

**Comment by saper** [comment]
rebuild

File: Makefile [hunk 3]
```diff
@@ -184,13 +184,9 @@ sassc: $(SASSC_EXE)
...
 
-$(DESTDIR)$(PREFIX)/:
-	$(MKDIR) $(DESTDIR)$(PREFIX)
-
-$(DESTDIR)$(PREFIX)/bin/:
-	$(MKDIR) $(DESTDIR)$(PREFIX)/bin
-
 $(DESTDIR)$(PREFIX)/bin/%: bin/%
+	-$(MKDIR) $(DESTDIR)$(PREFIX)
+	-$(MKDIR) $(DESTDIR)$(PREFIX)/bin
```
Comment (mgreter) [inline Makefile]: This change now fails under MSys2 mingw if directory already exists:
mkdir: cannot create directory ‘/usr/local/bin’: File exists
Makefile:188: recipe for target '/usr/local/bin/sassc.exe' failed

**Comment by saper** [comment]
So minus sign does not work? There should be no need to have targets for every directory we might possibly want to create?

**Comment by mgreter** [comment]
So far it has been the most reliable way to have a target for each directory to test if it already exists ...

**Pull request merged**